ambry-rest/src/main/java/com.github.ambry.rest/NettyResponseChannel.java

```diff
@@ -384,7 +384,7 @@ private FullHttpResponse getErrorResponse(Throwable cause) {
       restServiceErrorCode = ((RestServiceException) cause).getErrorCode();
       errorResponseStatus = ResponseStatus.getResponseStatus(restServiceErrorCode);
       status = getHttpResponseStatus(errorResponseStatus);
-      if (status == HttpResponseStatus.BAD_REQUEST) {
+      if (HttpStatusClass.CLIENT_ERROR.contains(status.code())) {
```

Comment:
what is the implication of returning info that is in exception messages in case of 403 and 401 (or other security related denials)?